tools/add_triage_signature.py

```diff
@@ -584,7 +584,7 @@ class AgentStepFailureSignature(Signature):
     )
 
     MAX_FAILURES_PER_HOST = 10
-    TRUNCATE_OUTPUT_LENGTH = 1500
+    TRUNCATE_OUTPUT_LINES = 100
```

Comment:
How much extreme length? :) It seems line wrap is supported, for moderate length at least.

Reply:
The reason I added the truncate in the first place because without truncating, the long line length of the install command froze my browser. I guess 1000 could be a good limit

Reply:
Oh this line is now split into multiple lines, so should be fine. But I can add some limitation just in case.